pkg/operator/controller/ca-configmap/controller.go

```diff
@@ -84,20 +84,20 @@ func New(mgr manager.Manager, config Config) (controller.Controller, error) {
 
 	// Watch the configmap from the source namespace
 	if err := c.Watch(
-		source.Kind(operatorCache, &corev1.ConfigMap{}),
-		&handler.EnqueueRequestForObject{},
-		predicate.And(predicate.NewPredicateFuncs(ctrlutils.InNamespace(config.SourceNamespace)), predicate.NewPredicateFuncs(ctrlutils.HasName(config.CAConfigMapName))),
-	); err != nil {
+		source.Kind[client.Object](operatorCache, &corev1.ConfigMap{},
+			&handler.EnqueueRequestForObject{},
+			predicate.And(predicate.NewPredicateFuncs(ctrlutils.InNamespace(config.SourceNamespace)), predicate.NewPredicateFuncs(ctrlutils.HasName(config.CAConfigMapName))),
```

Comment:
nit: was this manual or automatic?  If source.Kind changed this much, it might be better to put all the signature parameters on the same line.

Reply:
> was this manual or automatic? 

Manual, I had to do it myself. As far as I know, no migration tools were provided by `controller-runtime`.

> If source.Kind changed this much, it might be better to put all the signature parameters on the same line.

Yes, `source.Kind` is a generic function now. Wouldn't it be too lengthy in a single line?